### PR TITLE
docs: missing CRD awsnodetemplates.karpenter.k8s.aws annotations

### PR DESCRIPTION
**Pull request by yahavb**
Fixes # 

**Description**

**How was this change tested?**

*

**Does this change impact docs?**
- [x] Yes, PR includes docs updates
- [ ] Yes, issue opened: # 
- [ ] No

**Release Note**

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/v0.26.1/upgrade-guide.md [hunk 1]
```diff
@@ -41,6 +41,18 @@ Karpenter ships with a few Custom Resource Definitions (CRDs). These CRDs are pu
     ```
     helm upgrade --install karpenter-crd oci://public.ecr.aws/karpenter/karpenter-crd --version vx.y.z --namespace karpenter --create-namespace
     ```
+  * Upgrading from older karpenter version that did not include `awsnodetemplates.karpenter.k8s.aws` labels and annotations requires manual CRDs annotations before issuing `helm upgrade`.
```
Comment (jonathan-innis) [inline website/content/en/v0.26.1/upgrade-guide.md:44]: Do you mind making this update in the `preview` along with the `v0.26.1`, `v0.26.0` and the `v0.27.0` docs. The `preview` docs are the ones that we carry forward each time we have a release.

File: website/content/en/v0.26.1/upgrade-guide.md [hunk 2]
```diff
@@ -41,6 +41,18 @@ Karpenter ships with a few Custom Resource Definitions (CRDs). These CRDs are pu
     ```
     helm upgrade --install karpenter-crd oci://public.ecr.aws/karpenter/karpenter-crd --version vx.y.z --namespace karpenter --create-namespace
     ```
+  * Upgrading from older karpenter version that did not include `awsnodetemplates.karpenter.k8s.aws` labels and annotations requires manual CRDs annotations before issuing `helm upgrade`. 
+
+In the case of `invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"` execute:
```
Comment (jonathan-innis) [inline website/content/en/v0.26.1/upgrade-guide.md:46]: Can you put these details under the troubleshooting in the "Install" section and then link to this troubleshooting in the upgrade guide?

**Review by yahavb — COMMENTED** [review]
Added comments to 0.25.0, 0.26.1, 0.27.0 and preview upgrade-guide

**Comment by jonathan-innis** [comment]
> Added comments to 0.25.0, 0.26.1, 0.27.0 and preview upgrade-guide

This PR is still blocked on moving that information from the upgrade guide to the troubleshooting section and referencing those details in the upgrade guide. Also, I think this change only needs to be back-ported to v0.26.1 since that was when the change was first introduced.

File: website/content/en/v0.26.1/upgrade-guide.md (hunk 1 above)
Comment (yahavb) [reply website/content/en/v0.26.1/upgrade-guide.md]: removed from 0.25 and made changes in `v0.26.1`, `v0.27.0` and `preview`. Removed changes from `v0.25.0`

File: website/content/en/v0.26.1/upgrade-guide.md (hunk 2 above)
Comment (yahavb) [reply website/content/en/v0.26.1/upgrade-guide.md]: moved the changes to troubleshooting under Installation and referenced it in upgrade-guide

File: website/content/en/preview/troubleshooting.md [hunk 3]
```diff
@@ -63,6 +63,20 @@ If Helm is showing an error when trying to install Karpenter helm charts:
 - If you are getting a 403 forbidden error, you can try `docker logout public.ecr.aws` as explained [here](https://docs.aws.amazon.com/AmazonECR/latest/public/public-troubleshooting.html)
 - If you are receiving this error: `Error: failed to download "oci://public.ecr.aws/karpenter/karpenter" at version "0.17.0"`, then you need to prepend a `v` to the version number: `v0.17.0`. Before Karpenter moved to OCI helm charts (pre-v0.17.0), both `v0.16.0` and `0.16.0` would work, but OCI charts require an exact version match.
 
+
+### Helm Error when upgrading from older karpenter version
```
Comment (jonathan-innis) [inline website/content/en/preview/troubleshooting.md:67]:
```suggestion
### Helm Error when installing the `karpenter-crd` chart
```

Make sure to change this everywhere

File: website/content/en/preview/troubleshooting.md [hunk 4]
```diff
@@ -63,6 +63,20 @@ If Helm is showing an error when trying to install Karpenter helm charts:
...
+
+Upgrading from older karpenter version that did not include `awsnodetemplates.karpenter.k8s.aws` labels and annotations requires manual CRDs annotations before issuing `helm upgrade`.
```
Comment (jonathan-innis) [inline website/content/en/preview/troubleshooting.md:69]:
```suggestion
Karpenter v0.26.1+ introduced the `karpenter-crd` helm chart. When installing this chart on your cluster, if you have previously added the Karpenter CRDs to your cluster through the `karpenter` controller chart or through `kubectl replace`, Helm will reject the install of the chart due to `invalid ownership metadata`.
```

Make sure to change this everywhere

File: website/content/en/preview/upgrade-guide.md [hunk 5]
```diff
@@ -41,6 +41,8 @@ Karpenter ships with a few Custom Resource Definitions (CRDs). These CRDs are pu
     ```
     helm upgrade --install karpenter-crd oci://public.ecr.aws/karpenter/karpenter-crd --version vx.y.z --namespace karpenter --create-namespace
     ```
+* Follow [Troubleshooting](./troubleshooting.md#helm-error-when-upgrading-from-older-karpenter-version) when getting `invalid ownership metadata; label validation error:` after `helm upgrade` from older karpenter versions.
```
Comment (jonathan-innis) [inline website/content/en/preview/upgrade-guide.md:44]:
```suggestion
{{% alert title="Note" color="warning" %}}
Follow [Troubleshooting]({{<ref "./troubleshooting#helm-error-when-upgrading-from-older-karpenter-version" >}}) when getting `invalid ownership metadata; label validation error:` after `helm upgrade` from older karpenter versions.
{{% /alert %}}
```

The old link leads to a 404. Also, I think this callout should be good as a Note. Make sure to change this everywhere.

**Review by jonathan-innis — APPROVED** [review]
LGTM 🚀

**Pull request merged**